src/utils/redis/client.ts

```diff
@@ -0,0 +1,19 @@
+import { REDIS_PASSWORD, REDIS_HOST } from '@config';
+import { createClient, RedisClient } from 'redis';
+
+export const createRedisClient = (): RedisClient => {
+  const redisClient = createClient({
+    host: REDIS_HOST as string,
+    password: REDIS_PASSWORD as string
+  });
+  redisClient.on('connect', () => {
+    console.info(
```

Comment:
I think we should remove this
<img width="555" alt="Screen Shot 2021-06-24 at 22 47 04" src="https://user-images.githubusercontent.com/8084303/123356434-1a3fa780-d53e-11eb-8cf4-7074bec5e4ad.png">

Reply:
Good point, I removed the log when redis is connected.